identifiers/country-us/state-ar-local_gov.csv

```diff
@@ -262,11 +394,45 @@ ocd-division/country:us/state:ar/county:lawrence/council_district:6,Lawrence Cou
+ocd-division/country:us/state:ar/county:lee/council_district:8,Lee County District 8
+ocd-division/country:us/state:ar/county:lee/council_district:9,Lee County District 9
+ocd-division/country:us/state:ar/county:lincoln/council_district:1,Lincoln County District 1
+ocd-division/country:us/state:ar/county:lincoln/council_district:2,Lincoln County District 2
+ocd-division/country:us/state:ar/county:lincoln/council_district:3,Lincoln County District 3
+ocd-division/country:us/state:ar/county:lincoln/council_district:4,Lincoln County District 4
+ocd-division/country:us/state:ar/county:lincoln/council_district:5,Lincoln County District 5
+ocd-division/country:us/state:ar/county:lincoln/council_district:6,Lincoln County District 6
+ocd-division/country:us/state:ar/county:lincoln/council_district:7,Lincoln County District 7
+ocd-division/country:us/state:ar/county:lincoln/council_district:8,Lincoln County District 8
+ocd-division/country:us/state:ar/county:lincoln/council_district:9,Lincoln County District 9
+ocd-division/country:us/state:ar/county:little_river/council_district:1,Little River County District 1
+ocd-division/country:us/state:ar/county:little_river/council_district:2,Little River County District 2
+ocd-division/country:us/state:ar/county:little_river/council_district:3,Little River County District 3
+ocd-division/country:us/state:ar/county:little_river/council_district:4,Little River County District 4
+ocd-division/country:us/state:ar/county:little_river/council_district:5,Little River County District 5
+ocd-division/country:us/state:ar/county:little_river/council_district:6,Little River County District 6
+ocd-division/country:us/state:ar/county:little_river/council_district:7,Little River County District 7
+ocd-division/country:us/state:ar/county:little_river/council_district:8,Little River County District 8
+ocd-division/country:us/state:ar/county:little_river/council_district:9,Little River County District 9
+ocd-division/country:us/state:ar/county:logan/council_district:1,Logan County District 1
+ocd-division/country:us/state:ar/county:logan/council_district:2,Logan County District 2
+ocd-division/country:us/state:ar/county:logan/council_district:3,Logan County District 3
+ocd-division/country:us/state:ar/county:logan/council_district:4,Logan County District 4
+ocd-division/country:us/state:ar/county:logan/council_district:5,Logan County District 5
+ocd-division/country:us/state:ar/county:logan/council_district:6,Logan County District 6
+ocd-division/country:us/state:ar/county:logan/council_district:7,Logan County District 7
+ocd-division/country:us/state:ar/county:logan/council_district:8,Logan County District 8
+ocd-division/country:us/state:ar/county:logan/council_district:9,Logan County District 9
+ocd-division/country:us/state:ar/county:logan/council_district:10,Logan County District 10
+ocd-division/country:us/state:ar/county:logan/council_district:11,Logan County District 11
```

Comment:
For Logan County we followed the population-to-quorum court district table in [AR code 14-14-402](https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=437b8f7d-bce3-4470-9e49-2fed156bbeef&nodeid=AAOAACAACAAFAAD&nodepath=%2FROOT%2FAAO%2FAAOAAC%2FAAOAACAAC%2FAAOAACAACAAF%2FAAOAACAACAAFAAD&level=5&haschildren=&populated=false&title=14-14-402.+Number+of+districts.&config=00JAA2ZjZiM2VhNS0wNTVlLTQ3NzUtYjQzYy0yYWZmODJiODRmMDYKAFBvZENhdGFsb2fXiYCnsel0plIgqpYkw9PK&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A4WVD-MSP0-R03N-D4SN-00008-00&ecomp=8gf5kkk&prid=aa307def-01a9-4d00-9beb-36517f955d60) and the [2020 census population numbers](https://www.census.gov/library/visualizations/interactive/2020-population-and-housing-state-data.html) (21,131) to decide on 11 districts, since many of the local sources were sparse or outdated. We also downloaded shapefiles from the [AR GIS website](https://gis.arkansas.gov/product/justice-of-the-peace-districts/), which also indicated 11 districts.

Reply:
Thanks for clarifying